.github/workflows/ci.yml

```diff
@@ -165,7 +165,7 @@ jobs:
           target: ${{ matrix.target }}
 
       - name: Run Cargo Test
-        run: cargo llvm-cov --all-features --lcov --output-path lcov.info
+        run: cargo llvm-cov --all-features --lcov --output-path lcov.info -- --skip ".skip.md"
```

Comment:
ah, yeah, we have single test that we actually want to skip =)

changing ci is not a good option here because we want to also skip those tests when running locally.

Let's remove it from ci and move to execution_spec i.e. don't execute files with `.skip` there.

Reply:
--skip ".skip.md" skips all files with that filter.
when running locally we can run this command right.
without codecov
cargo test --test execution_spec -- --skip ".skip.md"

Reply:
> move to execution_spec i.e. don't execute files with `.skip` there.

how to do it there?
i think datatest-stable didnt provide any ways to skip.

Reply:
ok. i will see if possible with pattern

Reply:
Hi @meskill,
any idea of regex pattern such that we do not match the .skip.md files.
Thank you.

Reply:
just leave regex as is and ignore the execution it in your test handler

Reply:
in what the datatest-stable will show it as passed not skipped.

Reply:
So, its better to go go with ci change if we want to use test files skipping.

Reply:
--skip is the best way here i think.

Reply:
@meskill ?

Reply:
developers usually just call `cargo test` without additional options to run all tests. Introducing mandatory option for the call is inconvenient. 

just add additional logs from spec output about skipped test if you worry about misleading stats

Reply:
ok.

Reply:
so, we need Executionspec::filter_spec()

Reply:
then how do we deal with only case.

Reply:
i think the skip and only are for developmental purposes.
1. "only" is for running locally. such tests are not to push to codebase. we can reverage cargo test -- --test "only test name to run"
2. "skip" is to skip tests run locally as well as in ci because (sometimes we may want avoiding tests runs but still want to save it) for that we can use cargo test -- --skip "test pattern to skip"

Reply:
@meskill 
i think its not much effort to a developer.

Reply:
then how can we handle "only" as we are moving file by file.
previously, the logic is like
loads all specs and filter them out based on only and skip.
Which is not possible as we are moving test by test.

Reply:
we can handle the skip by printing skipped message.
but only is not like that.

Reply:
I think you understand.

Reply:
sure, let me wrap it for you:
- `only` is used for experimenting purposes when developer locally want to focus on specific tests.  We don't mark `only` any tests on remote, only locally. That will be done by `cargo test` options now.
- `skip` is used to mark tests that are currently not working and we push that annotations to remote so they are not run by default. For that we need something like it worked previously either with special annotations or special file naming.

Reply:
ok.